documents/tasks/mails.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework import views
+from rest_framework.request import Request
+from rest_framework.response import Response
+from trans import trans
+
+from core.mixins.storage import StorageMixin
+from core.permissions.cron_or_task import IsCronOrTask
+from documents.models import Document, DocumentSendLog
+from mails.utils import send_mail
+from orders.models import Order
+
+
+class MailDocumentsTask(StorageMixin, views.APIView):
+    permission_classes = (IsCronOrTask,)
+
+    def post(self, request: Request, order_id: str, email: str):
+        documents = Document.objects.filter(order_id=order_id)
+        order = Order.objects.get(id=order_id)
+        documents_to_send = [
+            document for document in documents if email in document.receivers_emails
+        ]
+
+        attachments_as_blobs = [
+            self.bucket.blob(document.blob_name) for document in documents_to_send
+        ]
+
+        send_log = DocumentSendLog.objects.select_for_update().filter(
+            order_id=order_id, email=email
+        )
+
+        if not DocumentSendLog.documents_sent(order_id=order_id, email=email):
+            send_mail(
```

Comment:
What happens if Mailgun returns an error here?

Reply:
The transaction will be rolled back and send log will still have `sent=False`